--- application.desktop.orig	2012-02-07 00:05:52.739258879 +0100
+++ qastools_0.17.1/qasmixer/share/application.desktop	2012-02-07 00:09:09.298169321 +0100
@@ -3,13 +3,15 @@
 Name=${PROGRAM_TITLE}
 GenericName=Sound volume mixer
 GenericName[de]=Klanglautstärkenregler
+GenericName[it]=Controllo del volume del suono
 Exec=${PROGRAM_NAME}
 Icon=${PROGRAM_NAME}
 StartupNotify=false
 Terminal=false
-Categories=AudioVideo;Audio;Mixer
+Categories=AudioVideo;Audio;Mixer;
 Comment=Graphical mixer application for the ALSA
 Comment[cs]=Grafický směšovač pro ALSA
 Comment[de]=Graphische Mixeranwendung für ALSA
 Comment[es]=Mezclador gráfico para ALSA
+Comment[it]=Mixer gráfico per ALSA
 Comment[ru]=Графический микшер для звуковой системы ALSA
